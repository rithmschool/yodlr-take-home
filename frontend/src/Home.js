import React from 'react';


const Home = () => (
  <div className="row justify-content-center mt-5">
    <div className="card col-sm-8">
      <div className="card-body">
        <h3>Welcome to Yodlr</h3>
        <h5><i>Please feel free to explore and learn more about what we can offer!</i></h5>
      </div>
    </div>
  </div>
)


export default Home;