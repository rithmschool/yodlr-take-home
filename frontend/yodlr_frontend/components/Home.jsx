import React from "react";

function Home() {
    return (
      <div className="App">
        <h1>Welcome to the User Interface</h1>
      </div>
    )
  }
  
  export default Home;
  