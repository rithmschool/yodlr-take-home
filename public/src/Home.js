import React from 'react';
import { Button } from 'reactstrap';

// display home page
function Home () {
    return(
        <div>
            <h3>Welcome to the Yodlr Design Challenge!</h3>
            <p>Click 'Sign Up' to register.</p>
            <Button color="primary" href="/signup">Sign Up</Button>
        </div>
    )
}

export default Home;