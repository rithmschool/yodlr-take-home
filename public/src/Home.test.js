import React from "react";
import { render } from "@testing-library/react";
import Home from './Home';

it("renders without crashing", function() {
  render(<Home />);
});

it("matches snapshot", function() {
  const {asFragment} = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});