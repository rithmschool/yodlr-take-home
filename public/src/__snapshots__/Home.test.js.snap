// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h3>
      Welcome to the Yodlr Design Challenge!
    </h3>
    <p>
      Click 'Sign Up' to register.
    </p>
    <a
      class="btn btn-primary"
      href="/signup"
    >
      Sign Up
    </a>
  </div>
</DocumentFragment>
`;
